sklearn/model_selection/_split.py

```diff
@@ -1974,10 +1953,7 @@ def check_cv(cv='warn', y=None, classifier=False):
         The return value is a cross-validator which generates the train/test
         splits via the ``split`` method.
     """
-    if cv is None or cv == 'warn':
-        warnings.warn(CV_WARNING, FutureWarning)
-        cv = 3
-
+    cv = 5 if cv is None else cv
```

Comment:
I think we usually use this pattern:

```
if cv is None:
    cv = 5
```

not very important

Reply:
I find this more readable. +0.

Reply:
I like `cv = cv or 5` :)

Reply:
`cv or 5` is bad if you have cv=0 or cv=object where object has `__len__(self): return 0`... but those are unlikely at least.